yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -125,11 +126,16 @@ func assessMigrationBulk() {
 	if err != nil {
 		utils.ErrExit("failed to generate bulk assessment report: %s", err)
 	}
+
+	err = generateBulkAssessmentReport(dbConfigs)
+	if err != nil {
+		utils.ErrExit("failed to generate bulk assessment report: %s", err)
+	}
```

Comment:
duplicate code, probably came during some merge conflict resolution.
Will remove it